### PR TITLE
Fix missing events in IE11 #16

### PR DESCRIPTION
**Pull request by dhilgarth**

### CODE REVIEW

File: lib/changes.js [hunk 1]
```diff
@@ -103,23 +103,19 @@ export function removeNodes(selectorObserver, changes, nodes) {
 //
 // Returns nothing.
 export function revalidateObservers(selectorObserver, changes, node) {
-  if ('matches' in node) {
```
Comment (josh) [inline lib/changes.js]: So one thing this guards against is `nodeType`s like `TEXT_NODE`s that don't implement `matches`.

**Comment by dhilgarth** [comment]
Thanks for the feedback. I have pushed a new commit that should fix this

**Comment by dhilgarth** [comment]
Great, thanks!

**Comment by josh** [comment]
CC: @dgraham @muan for additional 👀s

File: lib/changes.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ export function addNodes(selectorObserver, changes, nodes) {
   for (let i = 0; i < nodes.length; i++) {
     const node = nodes[i]
 
-    if ('matches' in node) {
+    if (supportsSelectorMatching(node)) {
```
Comment (josh) [inline lib/changes.js]: Checking if `node` is an `Element` might be acceptable here too. Let me do some testing.

Comment (josh) [reply lib/changes.js]: Ah, `document` might be the edge case in some of these code paths. For an example, document implements `querySelectorAll` but is not an `Element`.

**Pull request merged**